package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.11.8-beta.0",
+  "version": "0.11.9-dev.20241127210659",
```

Comment:
Hi! Forgive me, I can't review this, I has been wrapping up my work before going on vacation for the next two weeks. I'll defer to @ollyd here :)